packages/cli/src/client-config/client_config_generator_adapter.ts

```diff
@@ -49,7 +49,8 @@ export class ClientConfigGeneratorAdapter {
       version,
       outDir,
       format,
-      (message) => printer.log(message)
+      (message) => printer.log(message),
+      (message) => printer.log(message, LogLevel.DEBUG)
```

Comment:
With LogLevel being in `cli-core`, this was what I could think of to avoid importing LogLevel in the `client-config` package.